rspace/src/main/scala/coop/rchain/rspace/history/HistoryRepositoryImpl.scala

```diff
@@ -148,8 +148,8 @@ final case class HistoryRepositoryImpl[F[_]: Sync, C, P, A, K](
         s"${key.toHex};delete-join;0"
     }
 
-  private def transform(actions: List[HotStoreAction]): List[Result] =
-    actions.map {
+  private def transform(action: HotStoreAction): Result =
```

Comment:
here too

Reply:
looks like I didnt do a rebase. the change was introduced with the commit: 3a987cfa